features/projects/types/project.types.ts

```diff
@@ -1,31 +1,31 @@
 export enum ProjectLanguage {
-	react = 'react',
-	node = 'node',
-	python = 'python',
+  react = "react",
+  node = "node",
+  python = "python",
 }
 
 export enum ProjectStatus {
-	stable = 'stable',
-	warning = 'warning',
-	critical = 'critical',
-	error = 'error',
-	info = 'info',
+  stable = "stable",
+  warning = "warning",
+  critical = "critical",
+  error = "error",
+  info = "info",
```

Comment:
In my opinion, the enum should reflect the possible values in the API data. So I'd rather replace `stable` and `critical` with `info` and `error` instead of adding them. After all, `stable` and `critical` don't exist in the API data.

Can you think of a solution using this approach?